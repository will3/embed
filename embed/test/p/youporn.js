const test = require('../utils/test');

test('youporn',
  'https://www.youporn.com/watch/13747723/nubilefilms-passionate-threeway-makes-bff-squirt');
