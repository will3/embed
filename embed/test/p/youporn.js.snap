// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` youporn 1`] = `
Object {
  "description": "Watch Nubilefilms - Passionate Threeway Makes Bff Squirt online on YouPorn.com. YouPorn is the biggest HD porn video site with the hottest ffm movies!",
  "originalUrl": "https://www.youporn.com/watch/13747723/nubilefilms-passionate-threeway-makes-bff-squirt",
  "site_name": "youporn",
  "thumbnail": Object {
    "url": "https://fi1.ypncdn.com/m=eaAaaEPbaaaa/201704/28/13747723/original/6/nubilefilms-passionate-threeway-makes-bff-squirt-6.jpg",
  },
  "title": "Nubilefilms - Passionate Threeway Makes Bff Squirt - Free Porn Videos - YouPorn",
  "video": Object {
    "description": undefined,
    "height": "360",
    "title": undefined,
    "url": "https://www.youporn.com/embed/13747723/nubilefilms-passionate-threeway-makes-bff-squirt/",
    "width": "640",
  },
}
`;
